app/views/exercises/_editor.html.slim

```diff
@@ -39,9 +39,22 @@
             = t('exercises.editor.download')
 
       div
+        ruby:
+          if current_contributor.programming_group?
+            current_contributor.users.each do |user|
+              if user.id != current_user.id
+                @programming_partners_name = user.name
```

Comment:
If I am working with multiple partners, this variable is overwritten. Hence, only one of my partners is listed there.

Reply:
As discussed, it would be nice to show the status for larger groups as well. Potentially, there are three cases:

- All others are offline
- Some are online
- All others are online

We could add a hover tooltip to show the names.